app/build.gradle

```diff
@@ -1,4 +1,26 @@
+buildscript {
+    repositories {
+        maven { url 'https://maven.fabric.io/public' }
+    }
+
+    dependencies {
+        // These docs use an open ended version so that our plugin
+        // can be updated quickly in response to Android tooling updates
+
+        // We recommend changing it to the latest version from our changelog:
+        // https://docs.fabric.io/android/changelog.html#fabric-gradle-plugin
+        classpath 'io.fabric.tools:gradle:1.+'
```

Comment:
you can add it directly to our dependencies, there is no need for `buildscript` closure.
also, you can change the fabric tools version to: `1.25.3` instead of having 1.+
and remove the top maven source you have it below

Reply:
replace the version with : `1.25.3`